File: Day-16/Pattern24.py
n = int(input())

def PatternTwentyFour(n):
    i = 1
    while(i <=n):
        space = i -1
        while(space):
            print(' ', end='')
            space -=1
        
        num = n-i +1
        printNum = i
        while(num):
            print(printNum, end='')
            num-=1
            printNum+= 1

        i+=1
        print(end='\n')
    
PatternTwentyFour(n)

File: Day-17/Pattern24.js
function PatternTwentyFour(n) {
  let i = 1;
  while (i <= n) {
    let str = "";

    let spaces = i - 1;
    while (spaces) {
      str += " ";
      spaces--;
    }

    let nums = n - i + 1;
    let printNum = i;
    while (nums) {
      str += printNum;
      printNum++;
      nums--;
    }
    i++;
    console.log(str);
  }
}

PatternTwentyFour(4);
